git-tar/function/ops.go

```diff
@@ -461,3 +472,76 @@ func getShortSHA(sha string) string {
 	}
 	return sha[:7]
 }
+
+func formatTemplatePath(clonePath string) string {
+	folders := strings.Split(clonePath, "/")
+	return fmt.Sprintf("/%s", folders[1])
+}
+
+func copyTemplates(src, dest string) error {
+	info, err := os.Stat(src)
+	if err != nil {
+		return err
+	}
+	if info.IsDir() {
+		return copyDir(src, dest)
+	}
+	return copyFile(src, dest)
+}
+
+func copyDir(src, dest string) error {
+	info, err := os.Stat(src)
+	if err != nil {
+		return fmt.Errorf("error reading dest stats: %s", err.Error())
+	}
+
+	if err := os.MkdirAll(dest, info.Mode()); err != nil {
+		return fmt.Errorf("error creating path: %s - %s", dest, err.Error())
+	}
+
+	infos, err := ioutil.ReadDir(src)
+	if err != nil {
+		return err
+	}
+
+	for _, info := range infos {
+		if err := copyTemplates(
+			filepath.Join(src, info.Name()),
+			filepath.Join(dest, info.Name()),
+		); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func copyFile(src, dest string) error {
```

Comment:
Can the package be vendored from the faas-cli, or does it need to be duplicated and modified? cc @LucasRoesler @rgee0

Reply:
I totally stole the approach from you guys @rgee0 @LucasRoesler I think it should be vendored, but whatever you suggest I will go with it.

Reply:
The original code in the faas-cli was meant to be generic and reusable, if we can reuse it here, all the better